src/main/java/redis/clients/jedis/JedisFactory.java

```diff
@@ -35,6 +35,11 @@
     this(host, port, connectionTimeout, soTimeout, password, database, clientName,
         false, null, null, null);
   }
+  JedisFactory(final String host, final int port, final int connectionTimeout,
+               final int soTimeout, final String password, final int database, final String clientName, final boolean isRedisSslEnabled) {
```

Comment:
Do we need this constructor? Isn't following constructor enough?

I'm asking this because JedisFactory is kept to have fewest constructors.